<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThicknessDialog</class>
 <widget class="QDialog" name="ThicknessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="rt_thickness" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>334</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="penStyleComboBox">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>36</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>180</width>
        <height>30</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>SolidLine</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/SolidLine.png</normaloff>:/icons/images/SolidLine.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashLine</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/DashLine.png</normaloff>:/icons/images/DashLine.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DotLine</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/DotLine.png</normaloff>:/icons/images/DotLine.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashDotLine</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/DashDotLine.png</normaloff>:/icons/images/DashDotLine.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashDotDotLine</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/DashDotDotLine.png</normaloff>:/icons/images/DashDotDotLine.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QSlider" name="thicknessSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="thicknessNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>66</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ThicknessDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ThicknessDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
